bin/add_celltypes_to_sce.R

```diff
@@ -74,30 +93,22 @@ get_ref_info <- function(ref_filename, extension) {
   return(ref_info)
 }
 
-# check that input file exists
-if (!file.exists(opt$input_sce_file)) {
-  stop("Missing input SCE file")
-}
-
-# check that output file ends in rds
-if (!(stringr::str_ends(opt$output_sce_file, ".rds"))) {
-  stop("output sce file name must end in .rds")
-}
+# check that input file exists and output file ends in rds
+stopifnot(
+  "Missing input SCE file" = file.exists(opt$input_sce_file),
+  "output sce file name must end in .rds" = stringr::str_ends(opt$output_sce_file, ".rds")
+)
 
 # read in input files
 sce <- readr::read_rds(opt$input_sce_file)
 
 # SingleR results --------------------------------------------------------------
 
-if (!is.null(opt$singler_results)) {
-  if (!file.exists(opt$singler_results)) {
-    stop("Missing SingleR results file")
-  }
-
-  if (is.null(opt$singler_model_file)) {
-    stop("SingleR model filename must be provided")
-  }
+if (file.exists(opt$singler_results)) {
```

Comment:
I think it makes sense to just set this directly and then use it in both places?
```suggestion
has_singler <- file.exists(opt$singler_results)
if (has_singler) {
```